MobileWallet/Screens/Home/TransactionHistory/TransactionsTableViewController.swift

```diff
@@ -68,4 +70,36 @@ class TransactionsTableViewController: UITableViewController {
             sender.endRefreshing()
         })
     }
+
+    //Transaction gets tapped
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
```

Comment:
I prefer to use [`willSelectRowAt`](https://developer.apple.com/documentation/uikit/uitableviewdelegate/1614943-tableview) and return nil rather than this function and deselecting the row.